maintainers/maintainer-list.nix

```diff
@@ -9203,4 +9203,10 @@
     github = "deifactor";
     githubId = 30192992;
   };
+  btwiusegentoo = {
```

Comment:
Got a conflict because of this here, I suggest putting this in the alphabetically correct place instead because of this.

Reply:
Thanks for the suggestion. I fixed, rebased onto latest upstream and pushed